bedevere/prtype.py

```diff
@@ -43,7 +43,9 @@ async def classify_by_filepaths(gh, pull_request, filenames):
         if util.is_news_dir(filename):
             news = True
         filepath = pathlib.PurePath(filename)
-        if filepath.suffix == ".rst":
+        if filepath.suffix == ".rst" or (
+            util.is_docs_dir(filename) and filepath.name == ".nitignore"
+        ):
```

Comment:
We only have one file named this, and I don't expect we'll add others (if we do, we can deal with it then):

```suggestion
        if filepath.suffix == ".rst" or filepath.name == ".nitignore":
```